api/__tests__/routes/user.test.ts

```diff
@@ -209,10 +209,221 @@ describe("User tests", () => {
                 expectedResponse: user,
             });
         });
+
         test("DELETE /user/:id", async () => {
+            runner.authLevel(AuthenticationLevel.ADMINISTRATOR);
+            //we will create a new user and then delete him
+            const user = {
+                email: "foo@bar.com",
+                first_name: "Foo",
+                last_name: "Bar",
+                date_added: "2020-01-01T00:00:00.000Z",
+                last_login: "2020-01-01T00:00:00.000Z",
+                phone: "23457890",
+                address: {
+                    create: {
+                        city: "Gent",
+                        latitude: 90,
+                        longitude: 100.0,
+                        number: 1,
+                        street: "street",
+                        zip_code: 1000,
+                    },
+                },
+                student: false,
+                super_student: true,
+                admin: false,
+                password: "foobar",
+            };
+
+            const expectedUser = {
+                email: "foo@bar.com",
+                first_name: "Foo",
+                last_name: "Bar",
+                last_login: "2020-01-01T00:00:00.000Z",
+                date_added: "2020-01-01T00:00:00.000Z",
+                phone: "23457890",
+                address_id: 4,
+                student: false,
+                super_student: true,
+                admin: false,
+                deleted: false,
+                address: {
+                    id: 4,
+                    street: "street",
+                    number: 1,
+                    city: "Gent",
+                    zip_code: 1000,
+                    latitude: 90,
+                    longitude: 100,
+                },
+                regions: [],
+            };
+
+            await runner.post({
+                url: "/user",
+                data: user,
+                expectedResponse: expectedUser,
+            });
```

Comment:
Heel de bedoeling van lokale databank en diens reset na elke test is dat ge dit soort constructies niet hoeft te doen in de testen. 

Als je een user nodig hebt met specifieke velden / specifieke relaties met andere tabellen voor een test, kan je deze toevoegen in de initialisatie functie en niet in de test opzetten. Dit hoort hier niet thuis.